api/logs/logSender.go

```diff
@@ -87,14 +87,19 @@ func (ls *logSender) StartSendingBlocking() {
 	ls.doSendContinuously()
 }
 
-func (ls *logSender) waitForPatternMessage() error {
+func (ls *logSender) waitForProfile() error {
 	_, message, err := ls.conn.ReadMessage()
 	if err != nil {
 		return err
 	}
 
-	ls.log.Info("websocket log pattern received", "pattern", string(message))
-	err = logger.SetLogLevel(string(message))
+	ls.log.Info("websocket log profile received", "profile", string(message))
```

Comment:
I think we might improve this log level message (string on some particular bytes is too criptic).
Maybe move it on the line 101 and have something like:
```GO
ls.log.Info("websocket log profile received",
"pattern", profile.LogLevelPatterns,
"with correlation", profile.WithCorrelation,
"with logger name", profile.WithLoggerName,
)

Reply:
Fixed.